README.md

```diff
@@ -31,3 +31,60 @@ The version number of the "bleeding edge" in `master` is always a `-SNAPSHOT` ve
 
 These plugins are published to R3's [Artifactory](https://software.r3.com/artifactory/corda). More recently, they are also published to Gradle's own plugins
 repository and can be imported into your projects using Gradle's `plugins` DSL.
+
+The plugins themselves fall into two categories: those intended for developing CorDapps, and those which are primarily used to build Corda itself.
+
+### Plugins for CorDapp development.
+
+- [`net.corda.plugins.cordapp`](cordapp/README.md):
+This plugin packages your CorDapp classes into a single jar file, along with
+the contants of all of the jar's dependent jars. Dependencies which are added
+to Gradle's `cordaCompile`, `cordaRuntime` and `cordapp` configurations are
+excluded from this packaging. The final jar is then signed.
+
+- [`net.corda.plugins.cordformation`](cordformation/README.rst):
+This plugin provides `Cordform` and `Dockerform` Gradle tasks for creating
+test deployments of multiple Corda nodes and their CorDapps. It also invokes
+the Network Bootstapper over the deployment for you, and provides you with a
+`runnodes` script to boot it all up afterwards.
```

Comment:
Maybe to add partiall task declaration exmaple and mention. it needs to depend on jar task, e.g.:

The task (by convetion named `deployNodes`) should depened on `jar` task,  e.g.:
``
task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
[...]
}
``

Reply:
This is _supposed_ to be a brief summary only. I do also provide links to other documentation for each plugin.